heron/instance/src/java/com/twitter/heron/instance/bolt/BoltInstance.java

```diff
@@ -96,6 +97,14 @@ public BoltInstance(PhysicalPlanHelper helper,
   }
 
   @Override
+  public void update(PhysicalPlanHelper physicalPlanHelper) {
+    if (bolt instanceof IUpdatable) {
+      ((IUpdatable) bolt).update(physicalPlanHelper.getTopologyContext());
```

Comment:
This is something we should look into separately. The same is true for all other calls to `bolt` in this class (e.g.,  `bolt.prepare(..)`,  `bolt.cleanup(..)`,  `bolt.execute(..)`). None have exception handling or timeouts.